readme.txt

```diff
@@ -2,7 +2,7 @@
 Contributors: getpantheon, danielbachhuber, Outlandish Josh, jspellman, jazzs3quence
 Tags: authentication, SAML
 Requires at least: 4.4
-Tested up to: 6.3
+Tested up to: 6.4.3

```

Comment:
AFAIK, issue in #395 is the simplesaml upgrade, not the php version/CMS version, if we get that working we should be fine to take this all the way to latest, unless you've discovered something.

Reply:
yes, see https://github.com/pantheon-systems/wp-saml-auth/pull/402#discussion_r2087595948